digital-objects/omap/16-lung-codex/draft/raw/metadata copy.yaml

```diff
@@ -0,0 +1,129 @@
+title: >-
+  OMAP-7 Organ Mapping Antibody Panel (OMAP) for Multiplexed Antibody-Based
+  Imaging of Human Lung with Cell DIVE
+description: >-
+  OMAP-7 was designed for Cell DIVE imaging of Formalin-Fixed Paraffin-Embedded
+  (FFPE) lung tissue. Cell DIVE enables highly multiplexed single cell analysis
+  using antibody-based imaging (Gerdes, MJ. et al, 2013). The tissue slides were
+  prepared with a two-step antigen retrieval process (pH 6 and pH 9, as
+  described https://dx.doi.org/10.17504/protocols.io.bpwumpew. Antigens were
+  detected using either primary antibodies with fluorophore-labeled secondaries
+  or fluorescent dye conjugated antibodies (protocol can be found
```

Comment:
on Protocols.io (link) -- Please unify so all links to Protocols.io are formatted like in OMAP-9.

Reply:
this was early draft; not in final